README.rst

```diff
@@ -1208,37 +1206,27 @@ As as side effect, the tool will validate the files for format errors, also some
 ordering of the types of the steps.
 
 
-.. _Migration from 3.x.x:
+.. _Migration from 4.x.x:
 
-Migration of your tests from versions 3.x.x
+Migration of your tests from versions 4.x.x
 -------------------------------------------
 
-
-Given steps are no longer fixtures. In case it is needed to make given step setup a fixture
-the target_fixture parameter should be used.
-
+The templated steps should use step argument parsers in order to match the scenario outlines
```

Comment:
we should also mention that the values of the examples will not be injected as fixtures anymore

Reply:
@olegpidsadnyi ⬆️